.github/CHANGELOG.md

```diff
@@ -146,7 +159,7 @@
 
 This release contains contributions from (in alphabetical order):
 
-Jack Ceroni, Josh Izaac, Maria Schuld, Antal Száva, Nicola Vitucci
+Juan Miguel Arazzola, Jack Ceroni, Josh Izaac, Nathan Killoran, Maria Schuld, Antal Száva, Nicola Vitucci
```

Comment:
:sweat_smile: made it in

Reply:
Anyone who did code review who should be included? Tom, for instance

Reply:
Perfect :+1: Was going to leave a comment here asking who else was missing. Theo as well, and Alain.